### PR TITLE
Add plone5.1 support

### PR DESCRIPTION
**Pull request by busykoala**
This pull enables plone5.1 support for the package.

### CODE REVIEW

File: ftw/events/browser/mopage.py [hunk 1]
```diff
@@ -85,7 +87,8 @@ def __call__(self):
         return super(MopageEvents, self).__call__()
 
     def import_node_attributes(self):
-        attrs = {'export_time': self.normalize_date(DateTime())}
+        attrs = {'export_time': self.normalize_date(datetime.now(
+            timezone('Europe/Zurich')))}
```
Comment (maethu) [inline ftw/events/browser/mopage.py]: Isn't that still wrong?

File: sources.cfg [hunk 2]
```diff
@@ -3,3 +3,9 @@ extends = http://plonesource.org/sources.cfg
 extensions += mr.developer
 
 auto-checkout =
+    ftw.keywordwidget
+    ftw.publisher.core
+
+[branches]
+ftw.keywordwidget = plone-5.1.x
+ftw.publisher.core = master
```
Comment (maethu) [inline sources.cfg]: Please make a release

File: ftw/events/browser/mopage.py (hunk 1 above)
Comment (busykoala) [reply ftw/events/browser/mopage.py]: I fixed it, didn't see it before thanks!

File: sources.cfg (hunk 2 above)
Comment (busykoala) [reply sources.cfg]: I removed the packages now.

**Comment by busykoala** [comment]
As soon as https://github.com/4teamwork/ftw.keywordwidget/pull/33 was merged and has been released the tests for this package should technically pass.

**Comment by busykoala** [comment]
@maethu, could you please look at that?

**Comment by busykoala** [comment]
@maethu ping

**Pull request merged**